src/test/java/com/wire/xenon/StatefulEncryptionTest.java

```diff
@@ -0,0 +1,59 @@
+package com.wire.xenon;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.waz.model.Messages;
+import com.wire.bots.cryptobox.IStorage;
+import com.wire.xenon.assets.MessageText;
+import com.wire.xenon.backend.models.Qualified;
+import com.wire.xenon.crypto.CryptoDatabase;
+import com.wire.xenon.crypto.storage.JdbiStorage;
+import com.wire.xenon.models.otr.PreKey;
+import com.wire.xenon.models.otr.PreKeys;
+import com.wire.xenon.models.otr.Recipients;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.UUID;
+
+public class StatefulEncryptionTest extends DatabaseTestBase {
+    @Test
+    public void testAliceToAlice() throws Exception {
```

Comment:
There were no tests to actually see what encryption and decryption do.
Just 2-3 tests on the messageProcessor and 2 disabled tests on the DAO.
I have enabled DAO test and added this one because it seems really valuable to have at least.
Also the pipeline seems to support a Postgres test database already so it should work